jenkins/verify-chef.bat

```diff
@@ -45,4 +45,11 @@ SET PATH=C:\opscode\%PROJECT_NAME%\bin;C:\opscode\%PROJECT_NAME%\embedded\bin;%P
 
 REM ; Test against the appbundle'd Chef
 cd c:\opscode\%PROJECT_NAME%\embedded\apps\chef
+
+IF "%PIPELINE_NAME%" == "chef-13" (
+REM ; Running unit and functional tests
+call bundle exec rspec -r rspec_junit_formatter -f RspecJunitFormatter -o %WORKSPACE%\test.xml -f documentation spec/unit spec/functional
+) ELSE (
+REM ; Running unit tests 
 call bundle exec rspec -r rspec_junit_formatter -f RspecJunitFormatter -o %WORKSPACE%\test.xml -f documentation spec/unit
```

Comment:
Appveyor runs every run on a fresh VM.  Jenkins...  not so much - and it shares builders between ad-hoc, git and release runs.  We struggled during the chef-12.4 release windows to get a useful build out because stuff would constantly break.  I think we fixed a few of the offending tests (the group specs with unicode characters for example) but I don't entirely trust it yet.